app/models/user.rb

```diff
@@ -0,0 +1,14 @@
+class User < ApplicationRecord
+  before_save { email.downcase! }
+  validates :name, presence: true, length: { maximum: 50 }
+  VALID_EMAIL_REGEX = Settings.email_regex
+  validates :email, presence: true, length: { maximum: 255 }, format: { with: VALID_EMAIL_REGEX }, uniqueness: { case_sensitive: false }
+  has_secure_password
+  validates :password, presence: true, length: { minimum: 6 }
+
+  # Returns the hash digest of the given string.
+  def User.digest(string)
+    cost = ActiveModel::SecurePassword.min_cost ? BCrypt::Engine::MIN_COST : BCrypt::Engine.cost
+    BCrypt::Password.create(string, cost: cost)
+  end
```

Comment:
For this file
1. Remove white space after start and before end of curly brace
2. Move the constant to settings.yml
3. Remove comment
4. Maximum 80 character per line